.travis.yml

```diff
@@ -33,7 +33,10 @@ cache:
 before_script:
   - 'export DISPLAY=:99.0'
   - 'sh -e /etc/init.d/xvfb start'
-  - sleep 3
+  - 'sleep 3'
+
+script:
+  - 'npm run test'
```

Comment:
This line isn't needed; Travis will automatically run `npm test`

Reply:
Nice to know :)